src/core/public/overlays/sidecar/helper.ts

```diff
@@ -3,6 +3,7 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+import { MIN_SIDECAR_SIZE } from './components/resizable_button';
```

Comment:
This is a circual import references here, can we avoid this?

Reply:
Done, moved the `MIN_SIDECAR_SIZE` into `helper.ts` reasonably